dev-support/create-release/do-release.sh

```diff
@@ -83,6 +83,14 @@ function should_build {
 }
 
 if should_build "tag" && [ "$SKIP_TAG" = 0 ]; then
+  if [ -z "${YETUS_HOME}" ] && [ "${RUNNING_IN_DOCKER}" != "1" ]; then
+    declare local_yetus="/opt/apache-yetus/0.11.1/"
```

Comment:
nit: you can use the symlink provided at `/usr/local/opt/yetus` to get whatever yetus version the user already has installed.

Reply:
Oh, you're intentionally pinning the Yetus version. Too bad you can't use `YETUS_VERSION` here...